index.html

```diff
@@ -7,14 +7,141 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
-
     <!-- Add your markup here -->
 
+    <!-- Header and nav bar -->
+    <header id="header">
+        <h1 class="md-col-3">
```

Comment:
I think the structure from the [CSS](https://github.com/CodeYourFuture/responsive-grid-project/blob/master/css/grid.css) should be:

```html
<div class="container">
<div class="row">
<!-- put columns here -->
</div>
</div>
```